tests/namespace/test_interfaces.py

```diff
@@ -237,7 +237,7 @@ def test_if_type_portchannel(self):
         self.assertEqual(value0.data, 161)
 
     def test_getnextpdu_first_bp_ifindex(self):
-        oid = ObjectIdentifier(11, 0, 0, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 1,1004))
+        oid = ObjectIdentifier(11, 0, 0, 0, (1, 3, 6, 1, 2, 1, 2, 2, 1, 1, 8999))
```

Comment:
Is this change required? There is no new interface added in namespace mock_dbs, is that right?

Reply:
@stepanblyschak This comment is still applicable.

---
In reply to: [527836254](https://github.com/Azure/sonic-snmpagent/pull/169#discussion_r527836254) [](ancestors = 527836254)

Reply:
I have added Ethernet16 https://github.com/Azure/sonic-snmpagent/pull/169/files/a00e69e25dfc7c17f2554a625b57c447aff477c0#diff-fe4c39c260164f04bfe84f3d68d6f6a5a28850a840667a26aaba67a9fc1afacbR48.
Now, different asic dbs have different port table.

Reply:
I have added Ethernet16 https://github.com/Azure/sonic-snmpagent/pull/169/files/a00e69e25dfc7c17f2554a625b57c447aff477c0#diff-fe4c39c260164f04bfe84f3d68d6f6a5a28850a840667a26aaba67a9fc1afacbR48.
Now, different asic dbs have different port table.